authors/apps/articles/serializers.py

```diff
@@ -18,14 +19,17 @@ class CreateArticleSerializer(serializers.ModelSerializer):
     class Meta:
         model = Article
 
-        fields = ['title', 'description', 'body', 'author_id']
+        fields = ['title', 'description', 'body', 'author_id', ]
 
     def validate(self, data):
         author_id = data.pop('author_id', None)
         author = User.objects.get(id=author_id)
         data.update({'author': author})
-
         return data
 
     def create(self, validated_data):
+        read_time = validated_data.pop('read_time',None)
+        read_time = readtime.of_text(validated_data.get('body'),wpm = 265)
```

Comment:
Hi @NicholusMuwonge , this is good work. Well done. I just have a few things I've noticed with your implementation. Firstly, line 31 of this file is redundant because at this point in the creation of an article, the read_time is not yet in the **validated_data** dictionary. That dictionary contains the data returned from the **validate** method. Therefore this `read_time = validated_data.pop('read_time',None)` will always evaluate to `read_time = None`. In any case, I see that you are initializing the `read_time` on line 32, `read_time = readtime.of_text(validated_data.get('body'),wpm = 265)`, so I am wondering what the purpose of line 31 is.

Reply:
hey @richien I am still struggling a bit with jungle semantics. I see it works fine without line 31 and I have fixed it. Thanks for the pointer